regimes/it/categories.go

```diff
@@ -90,19 +92,24 @@ var categories = []*tax.Category{
 		},
 	},
 	{
-		Code:     TaxCategoryRA,
+		// IT: https://www.agenziaentrate.gov.it/portale/imposta-sul-reddito-delle-persone-fisiche-irpef-/aliquote-e-calcolo-dell-irpef
+		// EN: https://www.agenziaentrate.gov.it/portale/web/english/information-for-specific-categories-of-workers
+		Code:     TaxCategoryIRPEF,
 		Retained: true,
 		Name: i18n.String{
-			i18n.EN: "RA",
-			i18n.IT: "RA",
+			i18n.EN: "IRPEF",
```

Comment:
Makes a lot more sense classifying the different withholding tax types separately than bunching them under RA

Reply:
The names aren't obvious, but yes, there should hopefully be a proper "code" for each type of retained tax.